characteristic_notify/index.html

```diff
@@ -30,7 +30,7 @@
   <h1>Bluetooth Characteristic Read w/Notifications</h1>
   <p>This is a simple test to subscribe to characteristic value change notifications.
   </p>
-  <div id="have_bluetooth" style="visibility: visible;">
+  <div id="bluetooth_available" style="visibility: visible;">
```

Comment:
I wonder if it is worth it to adopt a static content generation tool like [11ty](https://www.11ty.dev/) to avoid having to repeat common boilerplate.

Reply:
On my todo list to look into.